cicd/migration_helpers.py

```diff
@@ -1,9 +1,11 @@
-import requests
 import logging
 import json
 import os
 import time
 
+import requests
+import zipfile
```

Comment:
Let's get `pathlib` in for file path manipulations

```suggestion
import zipfile
from pathlib import Path
```